src/main/webapp/github-scm-source.js

```diff
@@ -5,47 +5,23 @@ Behaviour.specify("input[name$=_configuredByUrlRadio]", 'GitHubSCMSourceRadioCon
         return;
     }
 
-    var getNthParent = function(e, n) {
-        while (n > 0) {
-            if (e.parentNode) {
-                e = e.parentNode;
-                n--;
-            } else {
-                return null;
-            }
-        }
-        return e;
-    };
-
-    // Todo: Replace with a query selector?
-    var findNeighboringDynamicInput = function(e) {
-        var inputTbody = getNthParent(e, 4 /*tbody > tr > td > label > input*/);
-        if (inputTbody) {
-            // input hidden is always in the 4th position
-            var hiddenBlock = inputTbody.childNodes[4].firstElementChild.firstElementChild;
-            return hiddenBlock
-        }
-    };
-
-    var neighboringDynamicInput = findNeighboringDynamicInput(e);
-    if (neighboringDynamicInput) {
+    var configuredByUrlInput = e.closest('div[name="source"]').querySelector('input[name="configuredByUrl"]');
```

Comment:
closest is polyfilled by jenkins core for IE 11

Reply:
Can we make these selectors more robust? maybe removing the `div`.

Reply:
would that be `*[name="source"]` ?